PlaidLink.js

```diff
@@ -78,47 +78,49 @@ export const PlaidLink = ({
 PlaidLink.propTypes = {
   // Required props
 
-  // Displayed once a user has successfully linked their account
-  clientName: PropTypes.string.isRequired,
-
-  // The Plaid API environment on which to create user accounts.
-  env: PropTypes.oneOf(['development', 'sandbox', 'production']).isRequired,
-
   // A function that is called when a user has successfully onboarded their
   // account. The function should expect two arguments, the public_key and a
   // metadata object.
   onSuccess: PropTypes.func.isRequired,
 
-  // The Plaid product(s) you wish to use, an array containing some of
-  // auth, identity, income, transactions, assets, liabilities, investments.
-  product: PropTypes.arrayOf(
-    PropTypes.oneOf([
-      'auth',
-      'identity',
-      'income',
-      'transactions',
-      'assets',
-      'liabilities',
-      'investments',
-    ]),
-  ).isRequired,
-
   // Optional props
 
   // The public_key associated with your account; available from
   // the Plaid dashboard (https://dashboard.plaid.com).
-  // Either publicKey or token is required.
+  // 
+  // [DEPRECATED] - instead, pass a Link token into the token field.
+  // Create a Link token with the /link/token/create endpoint.
   publicKey: props => {
     if (!props.publicKey && !props.token) {
       return new Error(`One of props 'publicKey' or 'token' is required`);
     }
-    if (typeof props.publicKey !== 'string') {
+    if (typeof props.publicKey !== 'string' && !props.token) {
       return new Error(
         `Invalid prop 'publicKey': Expected string instead of ${typeof props.publicKey}`,
       );
     }
```

Comment:
💅 A log message at runtime indicating that `publicKey` has been deprecate might help drive adoption.

Reply:
⚡